### DISCUSSION TITLE
Crowdin is back!

### DISCUSSION THREAD

**Discussion by ovflowd** [nodejs/nodejs.org#4994] [Announcements]
Hey folks 👋 I'm excited to announce that our [Crowdin](https://crowdin.com) integration is working again!

We've updated the list of available languages to match the currently offered languages here. If you don't see your language listed in Crowdin, feel free to request access to Crowdin, and we will be more than happy to have you translate the website into your native language 🎉

Crowdin is a powerful tool for translators to easily translate parts of the website with a 0-code solution and an elegant and newcomer-friendly environment.

Interested in contributing to the Nodejs.org website translation? Check out [our Crowdin Project](https://crowdin.com/project/nodejs-website).

---

cc @nodejs/i18n @nodejs/website

**Comment by zeke** [comment]
Nice! Thanks for taking the initiative, @ovflowd. This effort has changed hands a lot over the years, but I think Crowdin is a great product, and they've been very supportive of Node.js' localization effort over the years (especially @andrulko 💚 🇺🇦 ) 

I used to actively maintain the tools in the @crowdin-node org, but haven't had the time lately. If anyone wishes to help take ownership of that stuff, give me a ping and I can help facilitate.

Looking forward to seeing this get revived!
